### ISSUE TITLE
Changing display configuration breaks 'function' item

### ISSUE THREAD

**Issue by denniedegroot** [AdyRock/com.ady.button_plus#101]
**Homey**
- Model: 2023
- Homey Firmware Version: Latest
- App Version: 0.4.4
- Button+ Firmware Version: 1.11

**Describe the bug**
Changing for example the X coordinate of a item in the display configuration corrupts the 'function' item.
This happens after saving.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to display configurations
2. Change x coordinate of a item
3. Press save
4. Function of all items are set to undefined

**Comment by AdyRock** [comment]
It took me a while to get to the bottom of this issue but I think I have found the way to make it happen. Initially I was changing the x position and then hitting tab or clicking away and then saving the settings. In that scenario everything is fine, so i couldn't recreate the problem. However, one time I changed the x position and then directly hit Save Configuration and the problem occurred. Thank goodness I implemented the import / export :-)
So, I have now narrowed it down to a timing issue between the position sorting and the save, so all I have to do is figure out how to fix it ;-)
